### PR TITLE
feat: Kubernetes support to Automated Self Checkout

### PR DESCRIPTION
**Pull request by brian-intel**
## PR Checklist

- [x] Added label to the Pull Request for easier discoverability and search
- [x] Commit Message meets guidelines as indicated in the URL https://github.com/intel-retail/automated-self-checkout/blob/main/CONTRIBUTING.md
- [x] Every commit is a single defect fix and does not mix feature addition or changes
- [ ] Unit Tests have been added for new changes
- [x] Updated Documentation as relevant to the changes
- [ ] All commented code has been removed
- [ ] If you've added a dependency, you've ensured license is compatible with repository license and clearly outlined the added dependency.
- [ ] PR change contains code related to security
- [ ] PR introduces changes that breaks compatibility with other modules (If YES, please provide details below)

## Issue this PR will close

close: #**issue_number**

### CODE REVIEW

File: helm/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+#
+# Copyright (C) 2023 Intel Corporation.
```
Comment (antoniomtz) [inline helm/docker-compose.yml:2]:
```suggestion
# Copyright (C) 2024 Intel Corporation.
```

File: helm/Makefile [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Copyright © 2023 Intel Corporation. All rights reserved.
```
Comment (antoniomtz) [inline helm/Makefile:1]:
```suggestion
# Copyright © 2024 Intel Corporation. All rights reserved.
```

File: helm/Dockerfile.cam-sim [hunk 3]
```diff
@@ -0,0 +1,12 @@
+#
+# Copyright (C) 2023 Intel Corporation.
```
Comment (antoniomtz) [inline helm/Dockerfile.cam-sim:2]:
```suggestion
# Copyright (C) 2024 Intel Corporation.
```

File: helm/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (C) 2023 Intel Corporation.
```
Comment (antoniomtz) [inline helm/Dockerfile:2]:
```suggestion
# Copyright (C) 2024 Intel Corporation.
```

File: helm/kubernetes.md [hunk 5]
```diff
@@ -0,0 +1,184 @@
...
+``` 
+kompose -f docker-compose.yml convert -o kubernetes/
+```
+
+## Useful helm commands
+
+helm package charts/ -u -d .deploy
+helm package charts/
+helm repo index .
+helm repo index --url https://github.com/brian-intel/retail-use-cases .
```
Comment (antoniomtz) [inline helm/kubernetes.md:55]: do we want to update the github URL to the intel account?

File: helm/kubernetes.md [hunk 6]
```diff
@@ -0,0 +1,184 @@
...
+
+helm repo add dls https://brian-intel.github.io/retail-use-cases/
```
Comment (antoniomtz) [inline helm/kubernetes.md:57]: same for this one

**Review by antoniomtz — CHANGES_REQUESTED** [review]

**Review by antoniomtz — APPROVED** [review]

**Review by antoniomtz — APPROVED** [review]

File: helm/kubernetes.md [hunk 7]
```diff
@@ -0,0 +1,179 @@
...
+## Stop cluster
+
+```
+minikube stop -p multinode
+```
+
+## Troubleshooting
+
+If you get the **ErrImagePull** message while deploying the pods, it means the local docker images can't be accessed by the cluster nodes.
+You can restarting the cluster by running:
```
Comment (NeethuES-intel) [inline helm/kubernetes.md:165]:
```suggestion
You can restart the cluster by running:
```

**Review by NeethuES-intel — APPROVED** [review]
Minor change.

**Pull request merged**